### PR TITLE
MC-12119: Delete storage classes, pvs and pvcs

### PR DESCRIPTION
**Pull request by patrickspensieri**
#### Changes made
Document deletion of
- storage classes
- persistent volumes
- persistent volume claims

#### Related PRs
- [cloudmc-kubernetes-sdk](https://github.com/cloudops/cloudmc-kubernetes-sdk/pull/133)

### CODE REVIEW

File: source/includes/kubernetes/_k8_storageclasses.md [hunk 1]
```diff
@@ -128,4 +128,32 @@ Retrieve a storage class and all its info in a given [environment](#administrati
...
 | `provisioner` <br/>_string_           | The provsioner for the storage class                                                                                                        |
 | `reclaimPolicy` <br/>_string_         | The default volume reclaim policy for this storage class. You have a choice between `Reclaim` or `Delete`.                                  |
-| `volumeBindingMode` <br/>_string_     | The default volume binding model for this storage class. You have a choice between `Immediate` or `WaitForFirstConsumer`.                   |
\ No newline at end of file
+| `volumeBindingMode` <br/>_string_     | The default volume binding model for this storage class. You have a choice between `Immediate` or `WaitForFirstConsumer`.                   |
+
+<!-------------------- DELETE A storage class -------------------->
+#### Delete a storage class
+
+<!-- TODO is cluster_id required -->
```
Comment (patrickspensieri) [inline source/includes/kubernetes/_k8_storageclasses.md]: `cluster_id` is only required in the extensions

File: source/includes/kubernetes/_k8_persistentvolumeclaims.md [hunk 2]
```diff
@@ -236,3 +236,30 @@ Return value:
...
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "taskId": "1542bd45-4732-419b-87b6-4ea6ec695c2b",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>DELETE /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/persistentvolumeclaims/:id</code>
```
Comment (lamminade) [inline source/includes/kubernetes/_k8_persistentvolumeclaims.md:258]: should probably write that the `:id` at the end is required  // that its the id of the pvc to delete?

**Review by lamminade — APPROVED** [review]
just the one comment which applies to all of them - otherwise looks good

**Review by evanluc — APPROVED** [review]

File: source/includes/kubernetes/_k8_persistentvolumeclaims.md (hunk 2 above)
Comment (patrickspensieri) [reply source/includes/kubernetes/_k8_persistentvolumeclaims.md:258]: We don't have a [Path parameters](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#delete-storageclass-v1-storage-k8s-io) section in our docs for K8 and other plugins.
- the `:service_code` and `:environment` are taken care of by the href
- we have a "Required" section but that's used only for fields required in a json body

So right now the `:id` is implied, should we create an improvement issue to add a "Path parameters" section or something similar?

**Pull request merged**